src/components/NcUserBubble/NcUserBubble.vue

```diff
@@ -239,8 +263,23 @@ export default {
 			return this.url && this.url.trim() !== ''
 		},
 
+		route() {
+			return getRoute(this.$router, this.url)
+		},
+
 		isLinkComponent() {
```

Comment:
(just a note: Why was this called like this in the first place `isX` always sounds like a boolean condition :pensive: )